db/seeds/activities/community.rb

```diff
@@ -313,9 +313,9 @@
   activity.category = 'community'
   activity.provider = 'stem-learning'
   activity.self_certifiable = true
-  activity.description = 'Encourage young people to develop important life skills through <a href="https://teachcomputing.org/primary-enrichment">enrichment</a> and engage with the wider community in practical, enjoyable, and meaningful ways.'
+  activity.description = 'Encourage young people to develop important life skills through <a href="https://teachcomputing.org/secondary-enrichment">enrichment</a> and engage with the wider community in practical, enjoyable, and meaningful ways.'
   activity.public_copy_description = 'Encourage young people to develop important life skills through enrichment and engage with the wider community in practical, enjoyable, and meaningful ways.'
-  activity.public_copy_title_url = 'http://www.teachcomputing.org/primary-enrichment'
+  activity.public_copy_title_url = 'http://www.teachcomputing.org/secondary-enrichment'
   activity.self_verification_info = 'Please provide us with evidence of delivery'
 
   activity.programmes = [primary_certificate]
```

Comment:
this doesn't look right as you are changing urls to secondary for an activity that is in primary cert. pls check

Reply:
There is a new activity specifically for the primary certificate

Reply:
From DMs
> David Corking
>   [1:56 PM](https://storm.slack.com/archives/D03PFAM5ZQE/p1693572990127679)
> line 321 still loads the secondary one into the primary cert.
> 
> Caleb Owens
>   [1:58 PM](https://storm.slack.com/archives/D03PFAM5ZQE/p1693573096143979)
> Yes. That is intentional as some users may have competed and it should still be visable to them
> [1:58](https://storm.slack.com/archives/D03PFAM5ZQE/p1693573120946369)
> I’m not entirely convinced that array is used for community activities though
> [2:01](https://storm.slack.com/archives/D03PFAM5ZQE/p1693573311987879)
> I probably should explicitly add the secondary certificate to it though
>